pages/common/hledger-aregister.md

```diff
@@ -0,0 +1,20 @@
+# hledger aregister
+
+> Show the transactions and running balances in one account.
+> More information: <https://hledger.org/hledger.html#aregister>.
+
+- Show transactions and running balance in the `assets:bank:checking` account:
+
+`hledger aregister assets:bank:checking`
+
+- Show transactions and running balance in the first account named `*savings*`:
```

Comment:
```suggestion
- Show transactions and running balance in the first account named `savings`:
```
We can remove the * symbol if it isn't required, as we don't support word styling in our pages as it is used for placeholder and command name highlighting.